src/gui/guiFormSpecMenu.cpp

```diff
@@ -323,6 +318,98 @@ void GUIFormSpecMenu::parseContainerEnd(parserData* data)
 	}
 }
 
+void GUIFormSpecMenu::parseScrollContainer(parserData* data, const std::string &element)
+{
+	std::vector<std::string> parts = split(element, ';');
+
+	if (parts.size() >= 3) {
+		std::vector<std::string> v_pos  = split(parts[0], ',');
+		std::vector<std::string> v_geom = split(parts[1], ',');
+		std::string scrollbar_name = parts[2];
+		std::string direction = parts[3];
+		f32 multiplier = 0.1f;
+		if (parts.size() >= 5 && !parts[4].empty())
+			multiplier = stof(parts[4]);
+
+		MY_CHECKPOS("scroll_container", 0);
+		MY_CHECKGEOM("scroll_container", 1);
+
+		v2s32 pos = v2s32((stof(v_pos[0]) + pos_offset.X) * imgsize.X,
+			(stof(v_pos[1]) + pos_offset.Y) * imgsize.Y);
+		v2s32 geom = v2s32(stof(v_geom[0]) * imgsize.X, stof(v_geom[1]) * imgsize.Y);
```

Comment:
According to the "bla" button's position (example formspec), this scaling does not match the formspec coordinates. Or did I miss anything?

Reply:
This is the same as in #8524. 
That PR will very probably be merged before this one. Supporting the old system with new elements doesn't make sense imo. Modders that use a new element have `real_coordinates` available and will use it.